src/authorizations/authorizations.module.ts

```diff
@@ -2,14 +2,15 @@
 
 import { forwardRef, Module } from '@nestjs/common';
 
+import { AccountsModule } from '~/accounts/accounts.module';
 import { AuthorizationsController } from '~/authorizations/authorizations.controller';
 import { AuthorizationsService } from '~/authorizations/authorizations.service';
 import { IdentitiesModule } from '~/identities/identities.module';
 import { PolymeshModule } from '~/polymesh/polymesh.module';
 import { TransactionsModule } from '~/transactions/transactions.module';
 
 @Module({
-  imports: [PolymeshModule, TransactionsModule, forwardRef(() => IdentitiesModule)],
+  imports: [PolymeshModule, TransactionsModule, forwardRef(() => IdentitiesModule), AccountsModule],
```

Comment:
```suggestion
  imports: [PolymeshModule, TransactionsModule, AccountsModule, forwardRef(() => IdentitiesModule)],
```
(nit) I think it looks better having all forwardRef together